travis/minimesos

```diff
@@ -48,9 +55,9 @@ fi
 docker run --rm -v "${MINIMESOS_HOST_DIR}":"${MINIMESOS_HOST_DIR}" \
        -v /var/run/docker.sock:/var/run/docker.sock \
        -v /sys/fs/cgroup:/sys/fs/cgroup \
-       -v ${DOCKER_BIN}:/usr/bin/docker \
        -i \
        --env DOCKER_HOST_IP=${DOCKER_HOST_IP} \
+       --env MINIMESOS_OS="${MINIMESOS_OS}" \
```

Comment:
I copy-pasted the content from https://github.com/ContainerSolutions/minimesos/blob/0.13.0/bin/minimesos